p/scripts/extra.js

```diff
@@ -3,6 +3,23 @@
 /* globals context, openNotification, xmlHttpRequestJson */
 /* jshint esversion:6, strict:global */
 
+function fix_popup_preview_path() {
+	var links = document.getElementsByClassName('popup-preview-path');
+
+	for (var i = 0; i < links.length; i++) {
+		var link = links[i];
+
+		link.onclick = function (event) {
```

Comment:
```suggestion
		link.addEventListener('click', function (ev) {
```

`ev` because that's how it's done elsewhere in the file.

addEventListener shouldn't be needed in this instance, but I consider it better practice.

Reply:
Ah yes ? Why that (it's real question, as said, I'm not a really web developer).

Reply:
By the way, why Travis is telling me that
```
Functions declared within loops referencing an outer scoped variable may lead to confusing semantics. (link)
```
Not sure if it's an error, but I don't understand how it can be a problem, it's common to refer a variable defined on parent block, no ? How do you fix that ?

Reply:
If you're using .onclick you're overriding everything else that may have been set.

If you're using addEventListener, you can attach (and easily remove) multiple listeners as desired.

Reply:
Yes, okay. Good point, thanks.

Reply:
That being said, I had to revert the change. We need to cancel the event by returning false (to prevent the browser to actually redirect to the link), which is not possible with a listener.

Reply:
I beg your pardon?

Reply:
Oh, you mean that in the literal sense. As in "return false" doesn't do it.

https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault

Reply:
I tested, and the browser was redirecting me (the `return false` was still there), which is not the case when using the `onclick` (by the way, it's what other "hooked" `<a>` are doing in the script) 🤷‍♂️

Perhaps a behavior which is not-standard (I tested only with Safari…)

[Edit] This `preventDefault` seems interesting. I will take a look.

Reply:
I was probably confused between proper JS and jQuery. In any case preventDefault is the right way to do it.